src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: $white-accent-color;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    display: flex;
+    text-align: center;
+  }
+
+  &__link {
+    color: $main-accent-color;
```

Comment:
The variable `$main-accent-color` is used here, but it's not defined in this file. Ensure that `$main-accent-color` is defined in a global SCSS file or imported into this file to avoid compilation errors.